includes/jobs/GlobalNewFilesInsertJob.php

```diff
@@ -24,18 +25,20 @@ public function run() {
 		$dbw = GlobalNewFilesHooks::getGlobalDB( DB_PRIMARY );
 
 		$uploader = $uploadedFile->getUploader( File::RAW ) ??
-				MediaWikiServices::getInstance()->getActorStore()->getUnknownActor();
+				$services->getActorStore()->getUnknownActor();
+
+		$centralIdLookup = $services->getCentralIdLookup();
 
 		$dbw->insert(
 			'gnf_files',
 			[
-				'files_dbname' => $config->get( 'DBname' ),
+				'files_dbname' => WikiMap::getCurrentWikiId(),
 				'files_name' => $uploadedFile->getName(),
 				'files_page' => $config->get( 'Server' ) . $uploadedFile->getDescriptionUrl(),
 				'files_private' => (int)!$permissionManager->isEveryoneAllowed( 'read' ),
 				'files_timestamp' => $dbw->timestamp(),
 				'files_url' => $uploadedFile->getViewURL(),
-				'files_user' => $uploader->getName(),
```

Comment:
`files_user` has the `NOT NULL` constraint, so we have to give the database _something_. I'd say patch the constraint out in the new SQL patch in this PR.